db/migrate/20200813175255_set_team_fieldsets.rb

```diff
@@ -0,0 +1,53 @@
+class SetTeamFieldsets < ActiveRecord::Migration
+  def change
+    started = Time.now.to_i
+    RequestStore.store[:skip_notifications] = true
+    fieldsets = [
+      {
+        identifier: 'tasks',
+        singular: 'task',
+        plural: 'tasks'
+      }.with_indifferent_access,
+      {
+        identifier: 'metadata',
+        singular: 'metadata',
+        plural: 'metadata'
+      }.with_indifferent_access
+    ]
+    # Set fieldsets setting to be the default for all teams
+    Team.find_each do |t|
```

Comment:
Please add a `print '.'` here to make debugging easier

Reply:
Done